src/e2e/specs/dashboard.spec.ts

```diff
@@ -381,3 +381,65 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Footer`, () =>
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Navigation`, () => {
+  test.beforeEach(async ({ dashboardPage, page }) => {
+    await dashboardPage.open();
+
+    try {
+      await checkAuthState(page);
+    } catch {
+      console.log("[E2E_LOG] - No fxa auth required, proceeding...");
+    }
+  });
+
+  test("Verify that the navigation bar options work correctly", async ({
+    dashboardPage,
+    page,
+  }) => {
+    // link to testrail
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463568",
+    });
+    const goToHrefOf = async (aTag: Locator) => {
+      const href = await aTag.getAttribute("href");
+      expect(href).toBeTruthy();
+      await page.goto(href!);
+    };
+    //testrail's step 1
+    await dashboardPage.goToDashboard();
+    const settingsButton = page.locator(
+      "//nav[@Class='Shell_mainMenu__lHVfI']//a[text()='Settings']",
+    );
```

Comment:
@pdehaan Yeah, I was wondering if I should reuse those - I thought that the locator did not update on every render. So, my thought was there's merit in 'refinding' these locators. This would also catch any instance where an element would disappear for some reason. What do you think?

Reply:
I'm definitely thinking we should be using the existing locators in the page object model, but I'd have to try and get this stack working locally to verify or see if there is an easy way for me to clone the tests and run them against Stage environment without needing to set up the local database and full stack.